src/vs/workbench/common/editor/filteredEditorGroupModel.ts

```diff
@@ -45,28 +43,22 @@ abstract class FilteredEditorGroupModel extends Disposable implements IReadonlyE
 	isSticky(editorOrIndex: number | EditorInput): boolean { return this.model.isSticky(editorOrIndex); }
 	isActive(editor: EditorInput | IUntypedEditorInput): boolean { return this.model.isActive(editor); }
 
-	getEditors(order: EditorsOrder, options?: { excludeSticky?: boolean }): readonly EditorInput[] {
-		const editors = this.model.getEditors(order, options);
-		return editors.filter(e => this.contains(e));
-	}
-
 	findEditor(candidate: EditorInput | null, options?: IMatchEditorOptions | undefined): [EditorInput, number] | undefined {
 		const result = this.model.findEditor(candidate, options);
 		if (!result) {
 			return undefined;
 		}
-		const editor = firstOrDefault(result, undefined);
-		if (!editor || typeof editor === 'number') {
-			return undefined;
-		}
-		return this.contains(editor) ? result : undefined;
+		return this.filter(result[1]) ? result : undefined;
```

Comment:
https://github.com/microsoft/vscode/pull/193081/files/2ba566548be8ea8bdaf61a8e43d92fb922d73483#r1331080582

Reply:
Yes, sorry for that. My initial feedback was wrong, because this is actually a tupel type and not a normal array, we are good here in terms of strict null. There is only strict null issues for arrays where you do a random access such as `const foo = []; console.log(foo[1])`